tools/ci/build-openmc.sh

```diff
@@ -18,8 +18,8 @@ cp /usr/local/lib/libopenmc.so openmc_src/lib/.
 cp -r /usr/local/lib/cmake openmc_src/lib/.
 cp /usr/local/lib/libpugixml.a openmc_src/lib/.
 cp -r /usr/local/lib/pkgconfig openmc_src/lib/.
-cp -r /usr/local/share/openmc openmc_src/share/.
-cp -r /usr/local/share/man openmc_src/share/.
+#cp -r /usr/local/share/openmc openmc_src/share/.
+#cp -r /usr/local/share/man openmc_src/share/.
```

Comment:
We can make a good first issue for somebody to remove these. I think if they're unused, we should just delete them and not comment them out.